ntbs-service/Models/Entities/TransferAlert.cs

```diff
@@ -28,8 +29,11 @@ public TransferAlert()
             ErrorMessage = ValidationMessages.StringWithNumbersAndForwardSlashFormat)]
         [Display(Name = "Optional note")]
         public string TransferRequestNote { get; set; }
+        public override string CaseManagerFullName => CaseManager?.FullName ?? "";
         public override string Action => "Transfer request to your TB service";
         public override string ActionLink => RouteHelper.GetNotificationPath(NotificationId.GetValueOrDefault(), NotificationSubPaths.TransferRequest);
         public override bool NotDismissable => true;
+        public string TransferReasonString => TransferReason.GetDisplayName().ToString() + 

```

Comment:
`.ToString()` not necessary - and opens door slightly to null reference (but GetDisplay name **does** always return a non-null string in this instance).

string.Empty should be preferred to `""`. I believe the brackets are redundant - but think I might leave them in anyway.

Reply:
curious why "" is bad? I can see both reading better in different situations..

Reply:
Something that was impressed upon me by one of the most knowledgeable .NET devs I've experienced - and logically it reads that `string.Empty` is interned and `""` is not... 

However this appears to be quite an outdated reading - and the compiler is more than able to plug the gap and intern "" too.

Reply:
ah gotcha! Does sound like something early versions of a language would have picked up